backend-api/generate_env_file.sh

```diff
@@ -11,4 +13,5 @@ echo "Generating .env file for the $SAM_STACK stack"
 
 SELF=$(aws cloudformation describe-stacks --stack-name "$SAM_STACK" --query "Stacks[0].Outputs[?OutputKey=='PublicAPIGWBaseURL'].OutputValue" --output text)
 
-echo "SELF_PUBLIC=$SELF" > .env
\ No newline at end of file
+echo "SELF_PUBLIC=$SELF" > .env
+echo "PROXY_API_URL=https://proxy-$SAM_STACK.review-b-async.$ENV.account.gov.uk" >> .env
```

Comment:
Just checking we will actually use the prod prefix here? I assume we will as it's a proxy api url for testing

Reply:
I think your question conflicts with itself!

Reply:
We will not need this in Prod :)